src/components/ItaliaTheme/Search/Search.jsx

```diff
@@ -615,77 +614,87 @@ const Search = () => {
             </aside>
 
             <Col lg={9} tag="section" className="py-lg-5">
-              {searchResults.loadingResults ? (
-                <Spinner active />
-              ) : searchResults?.result?.items_total > 0 ? (
-                <div
-                  className="search-results-wrapper"
-                  role="region"
-                  id="search-results-region"
-                  aria-live="polite"
-                >
-                  <div className="d-block ordering-widget">
-                    <Row className="pb-3 border-bottom">
-                      <Col xs={6} className="align-self-center">
-                        <p className="d-none d-lg-block" aria-live="polite">
-                          {intl.formatMessage(messages.foundNResults, {
-                            total: searchResults.result.items_total,
-                          })}
-                        </p>
-                        <p className="d-block d-lg-none mb-0 text-right">
-                          {intl.formatMessage(messages.orderBy)}
-                        </p>
-                      </Col>
-                      <Col xs={6}>
-                        <SelectInput
-                          id="search-sort-on"
-                          value={
-                            sortOnOptions.filter((o) => o.value === sortOn)[0]
-                          }
-                          label={intl.formatMessage(messages.orderBy)}
-                          placeholder={
-                            sortOnOptions.find((o) => o.value === sortOn).label
-                          }
-                          onChange={(opt) => setSortOn(opt.value)}
-                          options={sortOnOptions}
-                          defaultValue={sortOnOptions.find(
-                            (o) => o.value === 'relevance',
-                          )}
-                        />
-                      </Col>
-                    </Row>
-                  </div>
-                  <Row>
-                    {searchResults?.result?.items?.map((item, index) => (
-                      <Col md={12} key={item['@id']} className="p-0">
-                        <SearchResultItem
-                          item={item}
-                          index={index}
-                          searchableText={searchableText}
-                          section={getSectionFromId(item['@id'])}
-                        />
-                      </Col>
-                    ))}
+              <div
```

Comment:
se capisco bene in questo modo si vede sempre la parte di ordinamento dei risultati e del numero totale dei risultati e il pezzettino con scritto 'Trovati x risultati' che inizialmente sarà 'Trovati 0 risultati' e intanto sotto c'è lo spinner che gira.. 
E' ancora fuorviante.
Farei vedere solo ed esclusivamente lo spinner

Reply:
ok, allora sarà diverso da quanto approvato per v3, che limita i layout shift. Ci tornero' quando ho di nuovo il via libera da Sara per lavorare a io-comune per questioni interne di budget, la lascio in draft

Reply:
non sapevo su v3 fosse già così. 
Rimettila pure aperta che a questo punto lo uniformiamo al v3, poi ci penseremo di sistemare quel 'Trovati 0 risultati' più avanti.